src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -90,6 +90,10 @@ public static class MachineOptions {
                 "1.0 means 1L Steam -> 1 EU. 0.5 means 2L Steam -> 1 EU.", "Default: 0.5" })
         public double multiblockSteamToEU = 0.5;
 
+        @Config.Comment({ "Nuclear Max Power multiplier for balancing purposes", "Default: 0.1" })
+        @Config.RangeDouble(min = 0, max = 10000)
+        public double nuclearPowerMultiplier = 0.1;
```

Comment:
Should pull out the nuclear reactor configs into their own section (so a new subclass).